### DISCUSSION TITLE
Crow Feather Ticket reward for 100 chests daily

### DISCUSSION THREAD

**Discussion by RezA7X** [sunflower-land/sunflower-land#2781] [General]
hi I think it is better to give some season tickets to the players who will reach 100 chest days.

**Comment by sabryder** [comment]
Love that idea👀🍻❤️
Reward for royal sunflower players

**Comment by javad626** [comment]
There will be at least one difference between me and someone who skips the steps

**Comment by Aminbtdds** [comment]
This would be great reward for those who are loyal players😍🍻❤
Also, this could be extra route for giving feathers to players which will result in wider feather distribution.
Great suggestion🌻❤

**Comment by woooolf4** [comment]
I think it's an interesting idea and it can make a difference between a loyal player and an ordinary player.

**Comment by MahdisSunflower** [comment]
This is a great offer for loyal and active players🚀😍

**Comment by brian-man** [comment]
I don't get where people derive loyalty from being consistent with the daily chest.  It's a somewhat indicator of active, except for the 2-step issue breaking the streak for many players.

Aside from that, people are already upset by the 3x streak impact on the season.  Adding 100x streak disparity seems even worse.

For example:
<https://github.com/sunflower-land/sunflower-land/discussions/2779>
> Feathers +1 from the daily chest. REMOVE x3 for Feathers

**Reply by Ario8260** [reply]
We don't want 100x feathers, but if someone has 100 days of daily chest open, as a loyal player they get some tickets

**Reply by brian-man** [reply]
I didn't say 100x feathers.  I said 100x streak.

Even 1 feather added to the streak problem is a bad idea IMO.

**Reply by Ario8260** [reply]
but there must be a difference between a loyal player and normal players

**Reply by brian-man** [reply]
First, you keep assuming "click the daily" mean "loyal". It does not.

Second, "must be" is incorrect.

Third, "loyal but new" players can't even have such a streak for 3 months so you're basically wanting an earlier player advantage.  Earlier players already have a number of advantages.  They don't need this proposed handout.

**Reply by Ario8260** [reply]
you're right 100 days is too much but 50 days is fine for everyone😉

**Comment by Ario8260** [comment]
very good idea for active Planers 🐦‍⬛🐦‍⬛🐦‍⬛

**Comment by ShaGhaW1998** [comment]
That's a Good idea 👍🏼

**Comment by MrSnorch** [comment]
Bad idea, because the update with the open chests streak was released on 16.05.2023 at about 03:00 UTC, many players opened a chest before that time and did not get the streak, so because of this people will have a difference of one day.

If the update had come out on time I would have had 85 days, not 84
![image](https://github.com/sunflower-land/sunflower-land/assets/104712725/8dde489e-6b8b-4fda-ab78-21d9871c65c7)
The time in the screenshot is 03:24 UTC 16.05.2023
![image](https://github.com/sunflower-land/sunflower-land/assets/104712725/9662eea0-b6d6-4e6b-b5fe-675a25ea2922)

**Comment by LTNGx39** [comment]
I like the idea of giving a colorful reward for a streak of 100 days, a x10 or something greater, but that these are feathers I do not consider fair for most users
